wesnoth-thot.po

```diff
@@ -2569,18 +2582,13 @@ msgstr ""
 
 #. [special_note]
 #: data/campaigns/The_Hammer_of_Thursagan/utils/abilities.cfg:5
-#, fuzzy
-#| msgid ""
-#| " The presence of this unit inspires own units next to it to deal more "
-#| "damage in combat, though this only applies to units of lower or equal "
-#| "level."
 msgid ""
 "The presence of this unit inspires own units next to it to deal more damage "
 "in combat, though this only applies to units of lower or equal level."
 msgstr ""
-" Přítomnost této jednotky inspiruje sousedící spřátelené jednotky, které "
-"díky tomu způsobují svými útoky větší poškození. To však platí pouze pro "
-"jednotky na nižší nebo stejné úrovni."
+" Přítomnost této jednotky inspiruje sousedící spřátelené jednotky, které tak "
```

Comment:
mám za to, že Wesnoth rozlišuje vlastní a spřátelené - už jsem opravoval chybu, kdy překlad lhal o ovlivňování sousedních jednotek

aniž bych tedy zkoumal tento případ, navrhuji přeložit originál přesně - tedy jenom moje jednotky, ne spojenecké
```suggestion
"Přítomnost této jednotky inspiruje vlastní sousedící jednotky, které tak "
```

Reply:
Asi je to tak. Teď jsem to hrál, ale neměl jasem šanci to ověřit.
A ona už se nedělá mezera před větami, co jdou do zvláštních schopností?

Reply:
no v originále není, tak jsem ji smazal

nevzpomínám si (snad kromě popisu tlačítek a názvů sloupců tabulek) na to, že by bylo třeba uchylovat se v členění textu od originálu; ale počítám, že z nás dvou jste pamětník (Wesnothu) spíše vy

snažím se všechny opravy hned ověřit ve hře, ale ještě nemám nainstalovanou vývojovou (1.15) verzi

můžete se na to podívat vy?

Reply:
Já to zkusím někde najít.
Před lety se to dělalo, protože ty jednotlivé věty se do odstavce "zvláštní rysy" u popisu jednotky prostě kombinovaly. A když tam nebyla mezera, tak mezi větami (tj. pokud měla jednotka dva a více zvláštních rysů) ty mezery chyběly.

Reply:
Tak ve stable verzi tam ty mezery jsou. Ale pro většinu jednotek to není uloženo v units, ale v help. V development verzi to vypadá, že se ty mezery již nepoužívají.